src/EXTRA-COMPUTE/compute_stress_mop.cpp

```diff
@@ -133,27 +133,28 @@ ComputeStressMop::ComputeStressMop(LAMMPS *lmp, int narg, char **arg) : Compute(
         which[nvalues] = ANGLE;
         nvalues++;
       }
-    } else if (strcmp(arg[iarg],"dihedral") == 0) {
-      for (i=0; i<3; i++) {
+    } else if (strcmp(arg[iarg], "dihedral") == 0) {
+      for (i = 0; i < 3; i++) {
         which[nvalues] = DIHEDRAL;
         nvalues++;
       }
     } else
-      error->all(FLERR, "Illegal compute stress/mop command");    //break;
+      error->all(FLERR, "Unknown compute stress/mop keyword {}", arg[iarg]);
 
     iarg++;
   }
 
   // Error checks:
 
-  // orthogonal simulation box
-  if (domain->triclinic != 0)
-    error->all(FLERR, "Compute stress/mop is incompatible with triclinic simulation box");
-
   // 2D and pressure calculation in the Z coordinate
   if (domain->dimension == 2 && dir == Z)
     error->all(FLERR, "Compute stress/mop is incompatible with Z in 2d system");
 
+  // orthogonal simulation box
+
+  if (domain->triclinic != 0)
```

Comment:
This error check would be better moved into `init()`. Currently, I can issue a `change_box all triclinic` command after the compute is defined and bypass this check in examples/packages/mop/in.compute_stress_mop.